File: backend/routes/auth.routes.js
const express = require("express");
const router = express.Router();

const authController = require("../controllers/auth.controller");
const { verificarToken } = require("../middleware/auth.middleware");

router.post("/registro", authController.registrar);
router.post("/login", authController.login);
router.get("/perfil", verificarToken, authController.perfil);

module.exports = router;

File: backend/routes/auth_routes.py
from fastapi import APIRouter
from controllers import auth_controller
from schemas.auth_schema import UsuarioRegistro, UsuarioLogin

router = APIRouter()


@router.post("/registro")
def registro(user: UsuarioRegistro):
    return auth_controller.registrar(user)


@router.post("/login")
def login(user: UsuarioLogin):
    return auth_controller.login(user)